src/block/tab-labels/edit.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames'
+import { InspectorTabs } from '~stackable/components'
+import {
+	BlockDiv,
+	useGeneratedCss,
+	Alignment,
+	Advanced,
+	CustomCSS,
+	Responsive,
+	CustomAttributes,
+	EffectsAnimations,
+	ConditionalDisplay,
+	Separator,
+	Transform,
+} from '~stackable/block-components'
+import {
+	withBlockAttributeContext,
+	withBlockWrapperIsHovered,
+	withQueryLoopContext,
+} from '~stackable/higher-order'
+// import { version as VERSION, i18n } from 'stackable'
+
+/**
+ * WordPress dependencies
+ */
+import { compose } from '@wordpress/compose'
+import { __ } from '@wordpress/i18n'
+import { dispatch, select } from '@wordpress/data'
+import { attributes } from '../tabs/schema'
+import { getBlockFromExample } from '@wordpress/blocks'
+
+export const checkOpen = i => {
+	if ( i === 1 ) {
+		return true
+	}
+	return false
+}
+
+const Edit = props => {
+	const {
+		className,
+		isSelected,
+	} = props
+
+	useGeneratedCss( props.attributes )
+
+	const {
+		insertBlock, updateBlockAttributes,
+	} = dispatch( 'core/block-editor' )
+
+	const {
+		getBlockRootClientId, getNextBlockClientId,
+	} = select( 'core/block-editor' )
+
+	const tabContent = getNextBlockClientId( props.clientId )
+	const tabsBlock = getBlockRootClientId( props.clientId )
+
+	const blockClassNames = classnames( [
+		className,
+		'stk-block-tab-labels',
+	] )
+
+	const handleClick = el => {
+		const dataTab = parseInt( el.target.getAttribute( 'data-tab' ), 10 )
+		const tabContentBlock = document.querySelector( `[data-block="${ tabContent }"]` )
+		const columns = tabContentBlock.querySelectorAll( '[data-type="stackable/column"]' )
+		columns.forEach( ( element, index ) => {
+			if ( dataTab === index + 1 ) {
+				element.style.display = 'flex'
+			} else {
+				element.style.display = 'none'
+			}
+		} )
+	}
```

Comment:
This implementation looks like it's written for vanilla JS for the frontend. You'll need to change this. As much as possible, we don't want to manipulate CSS directly here.

I think one approach can be to add a `className` or `data-*` attribute to the `BlockDiv` (block-level css) in `edit` in the Tab Labels block, then use some CSS in the editor to hide the corresponding Tab Content column.